pkg/controllers/webhookownerreference/controller.go

```diff
@@ -0,0 +1,81 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Is the `Copyright` statement missing intended on `v0.33.x`?

Reply:
Yeah, expected. We didn't join K8s SIGs org until v0.34.0 which is when we added this additional copyright.